addons/website_slides/static/src/js/slides.js

```diff
@@ -20,7 +20,7 @@ publicWidget.registry.websiteSlides = publicWidget.Widget.extend({
             // then return fix formate string else timeago
             var displayStr = '';
             if (datetimeObj && new Date().getTime() - datetimeObj.getTime() > 7 * 24 * 60 * 60 * 1000) {
-                displayStr = moment(datetimeObj).format('ll');
+                displayStr = luxon.DateTime.fromJSDate(datetimeObj).toFormat('DD');
             } else {
                 displayStr = moment(datetimeObj).fromNow();
```

Comment:
moment here

Reply:
there is no equivalent with luxon so you would have to write a function. However, .fromNow() is only used 2x in the solution.

Reply:
So I didn't convert this one while waiting to see what we do

Reply:
What about toRelative ?
https://moment.github.io/luxon/api-docs/index.html#datetimetorelative

This is not exactly the same as moment() but it should be good enough if we don't want to implement our own solution.

Reply:
nice